template-parts/content/content-page.php

```diff
@@ -38,16 +38,9 @@
 			<?php
 			edit_post_link(
 				sprintf(
-					wp_kses(
-						/* translators: %s: Name of current post. Only visible to screen readers */
-						__( 'Edit <span class="screen-reader-text">%s</span>', 'twentytwentyone' ),
-						array(
-							'span' => array(
-								'class' => array(),
-							),
-						)
-					),
-					get_the_title()
+					/* translators: %s: Name of current post. Only visible to screen readers */
```

Comment:
Minor, pre-existing, issue: translators comments should always end with a period.

Reply:
Good point. I took the opportunity and fixed it for all translators comments :+1: